### ISSUE TITLE
VALIDATION QUERY

### ISSUE THREAD

**Issue by ali4321143** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#205]
Thanks for sharing the code
I have a question since we use the cross entropy loss as the MAIN metric to determine the  model performance on that train and validation
my question is that do we use the TEACHER FORCING even in the VALIDATION phase?->we provide the true label at each input and then the model give the output against each input and then we compute the cross entropy loss?
